lib/evss/aws/reference_data/service.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require 'evss/jwt'
+
+module EVSS
+  module AWS
+    module ReferenceData
+      class Service < EVSS::Service
+        include Common::Client::Monitoring
+
+        configuration EVSS::AWS::ReferenceData::Configuration
+
+        def get_countries
+          raw_response = nil
```

Comment:
what's the reason for initializing as `nil`?

Reply:
I originally had `EVSS::PCIUAddress::CountriesResponse.new(raw_response.status, raw_response)` outside of the `with_monitoring` block.  Now that it's back in, this can be removed.